docs/feature-flags/feature-flags-logs.md

```diff
@@ -83,14 +84,16 @@ A `feature_flag.evaluation` event SHOULD be emitted whenever a feature flag valu
 
 **[2] `error.type`:** If and only if an error occurred during flag evaluation.
 
-**[3] `feature_flag.variant`:** A semantic identifier, commonly referred to as a variant, provides a means
+**[3] `feature_flag.evaluation.value_type`:** If and only if `value` field is supplied in the event body
```

Comment:
`value_type` could certainly be valuable independently of a `value`.

Making this required when the `value` is defined seems like the minimum, but this field also seems independently valuable. I suppose we could consider making it generally required, though I'm leaning against that:

We've emphasized `variant` over `value` for understanding the result of the evaluation, the implicit reasoning here being that you can reliably use the `variant` to understand the `value` that was returned. The same argument could be made for the `value_type`, I suppose.

Reply:
What I have is consistent in that it still emphasizes variant over value and value_type. I'm not sure what you could learn from value_type that isn't implied by variant and version? We could make it "required if [...] else optional" if you think it provides value on its own.

Reply:
My comment might have been unclear; I was agreeing with the proposal as written, but just pointing out the alternative for consideration.